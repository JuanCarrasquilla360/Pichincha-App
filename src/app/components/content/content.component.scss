.content {
    background-color:  #E0E0E0;  
    max-width: 100vw;
    margin: 0 auto;
    padding: 20px;
    min-height: calc(100vh - 80px); 
  }