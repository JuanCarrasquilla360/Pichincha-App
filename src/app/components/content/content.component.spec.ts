import { ComponentFixture, TestBed } from '@angular/core/testing';

import { ContentComponent } from './content.component';

describe('ContentComponent', () => {
  let component: ContentComponent;
  let fixture: ComponentFixture<ContentComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [ContentComponent]
    })
    .compileComponents();
    
    fixture = TestBed.createComponent(ContentComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });
  test('should match with snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });
});
