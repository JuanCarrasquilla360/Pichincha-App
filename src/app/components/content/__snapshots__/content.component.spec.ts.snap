// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentComponent should match with snapshot 1`] = `
<div
  id="root1"
>
  <div
    class="content"
  >
    <router-outlet />
    
  </div>
</div>
`;
